deployment/lib/cf-url-rewriter.js

```diff
@@ -0,0 +1,59 @@
+const aws = require('aws-sdk');
```

Comment:
Could this be in typescript instead of js?

Reply:
If original repo don't use TS. The  n I would suggest to create a issue to track.

Reply:
One of the other files in the pull request is in typescript.

Reply:
Thanks all for the feedback! The reason why I choose JS is that

1) the AWS console will guide any customers (no matter inside Amazon or not) to JS code when clicking through Lambda -> blueprint 

<img width="1894" alt="Screen Shot 2022-01-31 at 12 24 01 AM" src="https://user-images.githubusercontent.com/60111637/151761782-f4b33d4b-8b32-4481-bd7b-01c174879595.png">

<img width="1879" alt="Screen Shot 2022-01-31 at 12 25 38 AM" src="https://user-images.githubusercontent.com/60111637/151761876-0b633eb4-ec2b-442c-8ccf-d4d85ba0cbd5.png">

2) The AWS SDK doc will guide customer to js code https://docs.aws.amazon.com/lambda/latest/dg/with-s3-example.html

This is the reason why I am working in JS code. I can definitely research more to see if TS is supported in AWS Lambda function. Do you have personal dev experiemnt to have TS working with AWS Lambda? 

If yes, please share any code link.

If not, I am happy to research more. (I may need to check how the transpiling of TS works in such case)

Reply:
Checkout https://apoorv.blog/typescript-lambda-cdk/ as it is an example of adding typescript code as a lambda along with some bundling settings.

Reply:
handled in https://github.com/opensearch-project/opensearch-build/pull/1660